### DISCUSSION TITLE
x20HD incompatible with the new HDZero Freestyle V2 VTX.

### DISCUSSION THREAD

**Discussion by carguy84** [FrSkyRC/ETHOS-Feedback-Community#3082] [Q&A]
I updated the firmware on the VRX from HDZero and the V2 works but I can't get a video stream to work with the X20HD.  Not sure if there's some sort of logging that I could put up here to help.

Chip-

**Comment by carguy84** [comment]
Actually I heard from FrSky and it's the Nano 90 camera that isn't compatible, which is what comes with the new kit unfortunately.

**Comment by Danycola1987** [comment]
scusate se scrivo qui, per questo modello di x20 non esiste il firmware FLEX ? il firmware IL cosa vuol dire ? grazie mille

**Comment by tresjam** [comment]
I'm not familiar with HDzero, but isn't it possible to install an older firmware?
